examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter

```diff
@@ -3645,11 +3645,11 @@ server cluster AccountLogin = 1294 {
   }
 
   response struct GetSetupPINResponse = 1 {
-    char_string setupPIN = 0;
+    nullable char_string setupPIN = 0;
```

Comment:
Adding nullable seems to be a wire-backwards incompatibility.

Should we change spec instead? @bzbarsky-apple @chrisdecenzo

Reply:
Well, this is a wire-format-incompatible change, so... Probably change the spec at this point.

Reply:
Undid the setupPin nullable change. Will create a Spec PR instead for that. Changes in access remain.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7838 is the spec PR